glue_statistics/glue_statistics.py

```diff
@@ -0,0 +1,4005 @@
+import os
```

Comment:
Ideally this file should be split up a bit, e.g. into a file for tools, one for states, etc.